### ISSUE TITLE
Bug: SpecialRange options doesn't tell user max value in template yaml

### ISSUE THREAD

**Issue by beauxq** [ArchipelagoMW/Archipelago#1104]
### What happened?

In the SpecialRange option, introduced here https://github.com/ArchipelagoMW/Archipelago/pull/630
the template yaml doesn't tell users the maximum value.

This is what comes out in the template yaml:
```
  continues: # 
    #   documentation here
    #    
    # you can add additional values between minimum and maximum
    0: 0 # minimum value
    3: 50
    random: 0
    random-high: 0
    random-low: 0
    infinity: 0
```
There's no indication of what the maximum value is.

### What were the expected results?

It would be useful for the user to be able to see the maximum value here.

### Software

Website

**Comment by Berserker66** [comment]
@alwaysintreble possibly a result of your template change?

**Comment by alwaysintreble** [comment]
The only change I made regarding ranges was that the 3 random values it prepopulates won't print twice if one if them is defined as default. Are we sure this but didn't already exist?

**Comment by beauxq** [comment]
progression balancing:
```
  progression_balancing: # A system that can move progression earlier, to try and prevent the player from getting stuck and bored early.
    #    [0-99, default 50] A lower setting means more getting stuck. A higher setting means less getting stuck.
    # you can add additional values between minimum and maximum
    random: 0
    random-high: 0
    random-low: 0
    disabled: 0
    normal: 50
    extreme: 0
```
The doc string mentions the max value, but the maximum value wouldn't be there otherwise.

**Comment by beauxq** [comment]
It looks like if the min or max is **not** named, there is a comment on the number to say it is the min or max.
```
0: 0 # minimum value
```
But if the min or max is named, there's nothing giving that information (unless it happens to be in the option docstring).

**Comment by alwaysintreble** [comment]
That makes sense. Having the actual min and max with a comment plus the named options with the game values will cause an issue so I would consider this as intended. I propose we could have a comment that explicitly states the min and max values for the range. That way the doc string doesn't need to specify it but we don't have duplicate values.

**Comment by ThePhar** [comment]
Issue was fixed in #1164, so I'm going to close this issue now. :)
